composer.json

```diff
@@ -16,25 +16,24 @@
         "jms/metadata": "~1.1",
         "jms/parser-lib": "1.*",
         "phpcollection/phpcollection": "~0.1",
-        "doctrine/annotations": "1.*",
-        "doctrine/instantiator": "~1.0.3"
+        "doctrine/annotations": "^1.0",
+        "doctrine/instantiator": "^1.0.3"
     },
     "conflict": {
         "twig/twig": "<1.12"
     },
     "suggest": {
         "symfony/yaml": "Required if you'd like to serialize data to YAML format."
     },
-    "minimum-stability": "dev",
     "require-dev": {
         "twig/twig": "~1.12|~2.0",
         "doctrine/orm": "~2.1",
-        "doctrine/phpcr-odm": "~1.0.1",
-        "jackalope/jackalope-doctrine-dbal": "1.0.*",
+        "doctrine/phpcr-odm": "^1.3",
+        "jackalope/jackalope-doctrine-dbal": "^1.0",
         "propel/propel1": "~1.7",
         "symfony/yaml": "2.*",
         "symfony/translation": "~2.0",
-        "symfony/validator": "~2.0",
+        "symfony/validator": "~2.0.0|~2.1.0",
         "symfony/form": "~2.1",
         "symfony/filesystem": "2.*",
         "phpunit/phpunit": "~4.0"
```

Comment:
Well, this should be considered as a bug. Symfony 2.1 is dead since many years (2.3 will die in a few months and it was a LTS release while 2.1 was not).
Btw, this was another drawback of committing a lock file: as the lock file was forcing to use outdated libraries, this library is actually broken since years without the CI catching it...